magefiles/utils.go

```diff
@@ -230,3 +231,56 @@ func cleanupQuay(quayService quay.QuayService, quayOrg string) error {
 	}
 	return nil
 }
+
+func cleanupQuayTags(quayService quay.QuayService, organization, repository string) error {
+	workerCount := 10
+	tagsChan := make(chan []quay.Tag, workerCount)
+	var wg sync.WaitGroup
+
+	var errors []error
+
+	for i := 0; i < workerCount; i++ {
+		wg.Add(1)
+		go func(tagsChan <-chan []quay.Tag, wg *sync.WaitGroup) {
+			defer wg.Done()
+
+			for tags := range tagsChan {
+				for _, tag := range tags {
+					if time.Unix(tag.StartTS, 0).Before(time.Now().AddDate(0, 0, -7)) {
+						deleted, err := quayService.DeleteTag(organization, repository, tag.Name)
+						if err != nil {
+							errors = append(errors, err)
```

Comment:
could we add more details to the error? i.e. add info about the organization, repository and tag name.